cloud

```diff
@@ -965,21 +985,23 @@ function execute_command() {
             --smtp-password-secret="$smtp_password_secret" \
             --database="$database"
         sections_run "$sections" scheduler_deploy \
-            "$project" "$prefix" "$trigger_topic"
+            "$project" "$prefix" "$load_queue_trigger_topic" "$pick_notifications_trigger_topic"
         sections_run "$sections" submitters_deploy \
             "$project" "$new_topic" "${submitters[@]}"
     elif [ "$command" == "withdraw" ]; then
         sections_run "$sections" submitters_withdraw \
             "$project" "$new_topic" "${submitters[@]}"
-        sections_run "$sections" scheduler_withdraw "$project" "$prefix"
+        sections_run "$sections" scheduler_withdraw \
+            "$project" "$prefix" "$load_queue_trigger_topic"
         sections_run "$sections" cloud_functions_withdraw "$project" "$prefix"
         sections_run "$sections" firestore_withdraw \
             "$project" "$spool_collection_path"
         sections_run "$sections" secrets_withdraw \
             "$project" "$smtp_password_secret"
         sections_run "$sections" pubsub_withdraw \
             --project="$project" \
-            --trigger-topic="$trigger_topic" \
+            --load-queue-trigger-topic="$load_queue_trigger_topic" \
+            --pick-notifications-trigger-topic="$pick_notifications_trigger_topic" \
```

Comment:
Could you wrap this line to 79 characters max too? You can just remove the equals sign, and have the option value in a separate argument, that will be handled fine. I.e.:
```suggestion
            --pick-notifications-trigger-topic \
                "$pick_notifications_trigger_topic" \
```